package.json

```diff
@@ -128,26 +134,28 @@
     "storybook-addon-performance": "0.14.0",
     "style-loader": "2.0.0",
     "syncpack": "^5.6.10",
+    "tmp": "0.2.1",
     "ts-jest": "24.3.0",
     "ts-loader": "8.0.14",
     "tsconfig-paths-webpack-plugin": "3.5.1",
+    "tslib": "2.2.0",
     "typescript": "4.1.5",
     "webpack": "5.21.2",
     "webpack-cli": "4.3.1",
     "webpack-dev-server": "4.0.0-beta.0",
-    "tmp": "0.2.1",
-    "yargs": "13.3.2",
-    "@nrwl/workspace": "12.1.0",
-    "@nrwl/cli": "12.1.0",
-    "@nrwl/tao": "12.1.0"
+    "webpack-bundle-analyzer": "4.4.2",
```

Comment:
propagated packages from scripts/ to single version policy as they had chalk as a dep which was causing issues

```
"just-scripts": "1.3.1",
"webpack-dev-middleware": "4.2.0",    
"webpack-hot-middleware": "2.25.0",
```